src/selfplay/loop.cc

```diff
@@ -132,6 +132,41 @@ void SelfPlayLoop::SendTournament(const TournamentInfo& info) {
   res += " draw " + std::to_string(info.results[1][0]) + " " +
          std::to_string(info.results[1][1]);
   SendResponse(res);
+
+  // Compute player1's ELO relative to player 2.
+  auto wins = info.results[0][0];
+  auto losses = info.results[2][0];
```

Comment:
It seems you only use player 1 white wins and not all his wins:

- All player1 wins are: info.results[0][0]+info.results[0][1]
- info.results[0][0] is for player1 white wins and [0][1] for player1 wins as black
- The same applies for all other results.

Reply:
Thanks for finding that! Fixed it.